capture.py

```diff
@@ -91,6 +91,16 @@ def capture(camera=None,
     """
 
     camera = camera or "persp"
+    
+    # Fallback to default settings
+    if camera_options is None:
```

Comment:
Using the default settings that `capture` had as they were set in the dictionaries.